Average minutemen facility

Latentish opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
![2022 07 12-01 30 39](https://user-images.githubusercontent.com/55075747/178447123-e0e0a75f-2f66-4d51-812e-7f9bce2093c2.png)

Adds a large scale xenomorph themed ruin, taking place in an overrun minutemen facility.
<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Adds a nice way to give the players a challenge in exchange for a guaranteed xenomorph antag spawning egg, as well as various other loot.
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog
:cl:phoaly
add: Adds your average Minutemen facility.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

Latentish committed:
base shape

- Adds the base shape

Latentish committed:
Completely redoes the facility

-Titular

Latentish committed:
Small changes

-Mostly adds some loot

Ketrai commented:
Feels like I'm looking add cdda with all the road stuff and concrete. Neat! I'd ask for a neutered borer spawn too but the subtype isn't a thing yet until i get that done.

Latentish committed:
Fuckload of detailing

-Adds a lot of detailing and fills out most of the rooms.

Latentish committed:
Adds most of the remaining map

- Adds a buttload of detailing
- Adds more reward
- Adds enemies
- Adds corpses

Latentish committed:
Merge branch 'master' into Average-Minutemen-Facility

Latentish committed:
Adds moar

more map
